homeassistant/helpers/device_registry.py

```diff
@@ -49,6 +50,12 @@ class _DeviceIndex(NamedTuple):
     connections: dict[tuple[str, str], str]
 
 
+class DeviceEntryType(Enum):
```

Comment:
Device entries are not the only "entries" we have, there are at least config entries. We tend to add longish prefixes so that things can be imported without ambiguity/aliasing. For example, in config entry land we have `ConfigEntryState`, not `EntryState`.